io/io/test/RRawFile.cxx

```diff
@@ -0,0 +1,170 @@
+#include "RConfigure.h"
+#include "ROOT/RRawFile.hxx"
+
+#include <algorithm>
+#include <cstdio>
+#include <cstring>
+#include <fstream>
+#include <memory>
+#include <stdexcept>
+#include <string>
+#include <utility>
+
+#include "gtest/gtest.h"
+
+using namespace ROOT::Detail;
+
+namespace {
+
+class FileRaii {
```

Comment:
Each class and non-trivial function needs to have at least one line of doxygen documentation.  It took me a while to realize the purpose of this class (creating a temporary file).